### ISSUE TITLE
Installed without errors - but where is it running? No browser opens

### ISSUE THREAD

**Issue by fbauer-kunbus** [Pikurrot/whisper-gui#30]
I installed with whisper-gui.bat on my Win-11 machine without any errors - but when I start it this is the output:

Checking conda installation...
conda is installed.
Checking config file...
Activating environment whispergui...
Environment activated.
Checking for updates...
Failed to fetch updates. Check your internet connection.
Running Whisper-GUI...
Press any key to continue . . .

And now?
Now browser window opened, and I don't find the gradio GUI running at any of the usual ports (e.g. 7860)

What can go wrong here?

Ok ... looks like this project is already dead ...

**Comment by chrisangel666** [comment]
Yes, this project is no longer valid. It was working successfully before, but it is no longer valid recently. Please check the source code author. WhisperX has also been updated recently.

**Comment by fbauer-kunbus** [comment]
> Yes, this project is no longer valid. It was working successfully before, but it is no longer valid recently. Please check the source code author. WhisperX has also been updated recently.

Since this project is no longer working, how can I completely UNinstall everything that it installed on my system using the .bat file?

**Comment by Pikurrot** [comment]
> > Yes, this project is no longer valid. It was working successfully before, but it is no longer valid recently. Please check the source code author. WhisperX has also been updated recently.
> 
> Since this project is no longer working, how can I completely UNinstall everything that it installed on my system using the .bat file?

See here how to completely uninstall: https://github.com/Pikurrot/whisper-gui/issues/32#issuecomment-2566372214

**Comment by Pikurrot** [comment]
> Yes, this project is no longer valid. It was working successfully before, but it is no longer valid recently. Please check the source code author. WhisperX has also been updated recently.

Thanks for pointing it out. I will take a look whenever I have time these days :)

**Comment by Pikurrot** [comment]
Your were right @chrisangel666, WhisperX and several other libraries got updates and the ones I specify in the environment were now incompatible. I found a range of versions for all of them to work well together now, and everything should be working again.
